js_modules/dagit/packages/core/src/runs/RunUtils.tsx

```diff
@@ -26,13 +26,16 @@ export const RunsQueryRefetchContext = React.createContext<{
   refetch: () => void;
 }>({refetch: () => {}});
 
-export function useDidLaunchEvent(cb: () => void) {
+export function useDidLaunchEvent(cb: () => void, delay = 1500) {
```

Comment:
Adding a delay here makes demos run more smoothly because when you "queue" a run it, even locally, it takes a moment to start. Refetching too fast means you don't see the "run in progress" notice